tests/Behat/TextFormatContext.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_editorial\Behat;
+
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Behat\Gherkin\Node\TableNode;
+use Drupal\filter\Entity\FilterFormat;
+use Drupal\user\Entity\Role;
+
+/**
+ * Defines step definitions for testing text formats.
+ */
+class TextFormatContext extends RawDrupalContext {
+
+  /**
+   * The IDs of text formats created during tests.
+   *
+   * @var array
+   */
+  protected $newTextFormats = [];
+
+  /**
+   * Deletes text formats created through the scenario.
+   *
+   * @param \Behat\Behat\Hook\Scope\AfterScenarioScope $scope
+   *   The scenario scope.
+   *
+   * @afterScenario @cleanup-formats
+   */
+  public function cleanupCreatedUrlAliases(AfterScenarioScope $scope): void {
+    $basic_roles = [
+      'anonymous',
+      'authenticated',
+    ];
+    foreach ($this->newTextFormats as $text_format_id) {
+      foreach ($basic_roles as $role) {
+        $role_object = Role::load($role);
+        $role_object->revokePermission('use text format ' . $text_format_id);
+        $role_object->save();
+      }
+      $text_format = FilterFormat::load($text_format_id);
+      $text_format->delete();
+    }
+  }
+
+  /**
+   * Checks the given node with title has the specified version numbers.
+   *
+   * // phpcs:disable
```

Comment:
If you have reasonable reason for disabling CS for this part of code, probably better to disable specific functionality of code sniffer.

Reply:
The @Then should not have any examples.
If you want to put an example, put it just under the "Checks the given node..." and yes it needs to be wrapped in full phpcs:disable like now.

Reply:
This needs to be fixed.

Reply:
Fixed